src/bitcoin-cli.cpp

```diff
@@ -64,18 +64,14 @@ static bool AppInitRPC(int argc, char* argv[])
     //
     ParseParameters(argc, argv);
     if (argc<2 || mapArgs.count("-?") || mapArgs.count("-h") || mapArgs.count("-help") || mapArgs.count("-version")) {
-        std::string strUsage = _("Zcash RPC client version") + " " + FormatFullVersion() + "\n";
+        std::string strUsage = _("Zcash RPC client version") + " " + FormatFullVersion() + "\n" + LicenseInfo();
```

Comment:
Why include this here?

- It's a sizeable inclusion, particularly for `zcash-cli`.
- It almost completely removes the need for `zcash-cli --version`.
- I'm pretty sure this will break `help2man`, or at least cause issues for it during the release process.